kernel/kernel-api/src/main/java/io/delta/kernel/internal/SnapshotImpl.java

```diff
@@ -169,6 +177,11 @@ public SnapshotReport getSnapshotReport() {
     return lazySnapshotReport.get();
   }
 
+  /** @returns the physical clustering columns in this snapshot */
```

Comment:
are clustering columns always phsical? would `getPhysicalClusteringColumns` be a more precise and less error-prone API? just to help enforce/remind consumers of the semantic?